corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -34,12 +34,10 @@ def get_commcare_users_by_filters(domain, user_filters, count_only=False):
     """
     role_id = user_filters.get('role_id', None)
     search_string = user_filters.get('search_string', None)
+    if not role_id and not search_string and not count_only:
+        return get_all_commcare_users_by_domain(domain)
+
     query = UserES().domain(domain).mobile_users()
-    if not role_id and not search_string:
-        if count_only:
-            query.count()
```

Comment:
previously this didn't return and so fell through to a similar if statement below which returned the count. Decided to remove this entirely and let the other one return

Reply:
good find